### ISSUE TITLE
Clicknload: /flash/add returns Error: 403 Forbidden

### ISSUE THREAD

**Issue by cinatic** [pyload/pyload#3700]
Hi,

first thanks for pyload, i am using it for a while now and it works really well. Since some days, i was not able to figure out what was the cause, clicknload shows only a 403 page anymore:

![Screenshot from 2020-08-14 20-34-28](https://user-images.githubusercontent.com/2721787/90281946-ce2bcd00-de6d-11ea-83b4-84c25e0c45bc.png)

Pyload is running as a docker container with the latest writl image https://hub.docker.com/r/writl/pyload and i am using chrome browser extension click'n'load proxy forwarder. 

Can you please help me to find the root cause?

**Comment by GammaC0de** [comment]
/flash/add accepts only POST method from 127.0.0.1 or localhost

**Comment by cinatic** [comment]
Wow wrf mind-blowing, how it was working for me in the past, I used it quiet often in the last weeks.

I guess it is the locale check decorator
https://github.com/pyload/pyload/blob/e24174964ddaf4b6972c9cd5d8856d8a976fa719/src/pyload/webui/app/blueprints/cnl_blueprint.py#L46

Either it is a modification in the docker image I used or I somehow changed the request headers. Would it be possible to make it work without the local port forward? Would it be possible to remove the decorator or are there more places where it could fail?
